generator/graphql-kotlin-schema-generator/src/main/kotlin/com/expediagroup/graphql/generator/internal/extensions/kParameterExtensions.kt

```diff
@@ -24,10 +24,6 @@ import kotlin.reflect.full.isSubclassOf
 
 internal fun KParameter.isInterface() = this.type.getKClass().isInterface()
 
-internal fun KParameter.isList() = this.type.getKClass().isSubclassOf(List::class)
-
-internal fun KParameter.isListType() = this.isList() || this.type.getJavaClass().isArray
```

Comment:
Since we are operating on the `KType` now in FunctionDataFetcher we can move these extensions to simplify